torchio/data/image.py

```diff
@@ -647,6 +647,38 @@ def plot(self, **kwargs) -> None:
             from ..visualization import plot_volume  # avoid circular import
             plot_volume(self, **kwargs)
 
+    def show(self, viewer_path: Optional[TypePath] = None) -> None:
+        """Open the image using external software.
+
+        Args:
+            viewer_path: Path to the application used to view the image. If
+                ``None``, the value of the environment variable
+                ``SITK_SHOW_COMMAND`` will be used. If this variable is also
+                not set, TorchIO will try to guess the location of
+                `ITK-SNAP <http://www.itksnap.org/pmwiki/pmwiki.php>`_ and
+                `3D Slicer <https://www.slicer.org/>`_.
+
+        Raises:
+            RuntimeError: If the viewer is not found.
+        """
+        sitk_image = self.as_sitk()
+        image_viewer = sitk.ImageViewer()
+        if viewer_path is not None:
+            image_viewer.SetApplication(str(viewer_path))
```

Comment:
Following from [this](https://discourse.slicer.org/t/loading-image-and-corresponding-mask-on-slicer-from-command-line/9930/2) thread, maybe we could consider using something like:

```py
if self.type == 'label':
    image_viewer.SetFileExtension('.label.nii')
```

This will ensure that the `LabelMap` will actually get loaded as a label map in Slicer, rather than as a normal image volume and should have no side-effect for other software. Thoughts?

Reply:
Slicer looks for the `.label` or `.seg` to the extension to try and identify label maps/segmentations when automatically loading from the command-line.

Reply:
Good idea! It doesn't seem to work with `label`, but I tried `seg.nrrd` and it's ok.